x/precisebank/keeper/invariants.go

```diff
@@ -0,0 +1,253 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/evm/x/precisebank/types"
+
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// RegisterInvariants registers the x/precisebank module invariants
+func RegisterInvariants(
+	ir sdk.InvariantRegistry,
+	k Keeper,
+	bk types.BankKeeper,
```

Comment:
I think `bk types.BankKeeper` is unused

Reply:
I removed it.
ref. https://github.com/cosmos/evm/pull/69/commits/783f5407f6c174cfc4356e205039aac524f6bacd